sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorExporterOptions.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace OpenTelemetry.Exporter.AzureMonitor
+{
+    public class AzureMonitorExporterOptions : ClientOptions
+    {
+        private const int DefaultCapacityKiloBytes = 50 * 1024;
+        private const long StorageCapacity = DefaultCapacityKiloBytes * 1024;
+
+        public string ConnectionString { get; set; }
```

Comment:
public setters gives you the problem where construction of the option doesn't enforce the restrictions/validation of the fields. So you end up with fun things like missing connection strings when they shouldn't be gone or someone changing it unexpectdely.